lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -190,6 +190,27 @@ def host_reachable?(host, what)
         true
       end
 
+      def host_resolvable?(host)
+        require 'resolv'
+
+        say("Checking if #{host} is resolvable ... ")
+        begin
+          ip_address = Resolv.getaddress(host)
+          if ip_address == "127.0.0.1" || ip_address == "::1"
```

Comment:
This is probably good enough, technically a link local address is a whole range of addrs `127.0.0.1/8` / `::1/128`

We could use IPAddr to do the subnet check, e.g.`IPAddr.new("127.0.0.1/8") === IPAddr.new("127.0.1.2")`

Reply:
```suggestion
          if IPAddr.new("127.0.0.1/8").include?(ip_address) || IPAddr.new("::1/128").include?(ip_address)
```